dnfdaemon-server/dbus/dnfdaemon-server.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=DNF daemon
+
+[Service]
+Type=dbus
+BusName=org.rpm.dnf.v0
+User=root
+ExecStart=/usr/bin/dnfdaemon-server
```

Comment:
Hm, I might be missing something but it seems the files in the `dnfdaemon-server` directory aren't used? Is this a working PoC? (I haven't tried to run it :grin:)